apps/OboeTester/app/src/main/cpp/jni-bridge.cpp

```diff
@@ -504,6 +518,26 @@ Java_com_mobileer_oboetester_OboeAudioStream_getSessionId(
     return result;
 }
 
+JNIEXPORT jstring JNICALL
+Java_com_mobileer_oboetester_OboeAudioStream_getPackageName(
+        JNIEnv *env, jobject, jint streamIndex) {
+    std::shared_ptr<oboe::AudioStream> oboeStream = engine.getCurrentActivity()->getStream(streamIndex);
+    if (oboeStream != nullptr) {
+        return env->NewStringUTF(oboeStream->getPackageName().c_str());
+    }
+    return (jstring) "";
```

Comment:
Is this enough or do you need to call NewStringUTF("")?

Reply:
This is dangerous as it directly to create a jstring via a C/C++ style string, it is likely to cause crash or undefined behavior when JAVA side receive it.